{{cookiecutter.github_repo}}/.travis.yml

```diff
@@ -1,6 +1,49 @@
-# deploy:
-#   provider: pypi
-#   user: $PYPI_USER
-#   password: $PYPI_PASSWORD
-#   on:
-#     tags: true
+language: python
+python:
+  - "3.6"
+services:
+  - docker
+
+jobs:
+  include:
+
+    - stage: Test "Development" Docker Image
+      if: tag IS present
+      script: scripts/build-run/build.sh -d
+        # (_identical_ to production build except for -e
+      # local directory package install indstead of PyPI.org)
+
+    # - stage: GitHub Release
```

Comment:
Are the GitHub and PyPI stages meant to be disabled permanently, or is this just a temporary phase while the cookiecutter stabilizes?

Reply:
@nam20485 is correct here, plan is to readd in a next update, so want to keep the reference